Podcast/LoginViewController.swift

```diff
@@ -71,7 +71,8 @@ class LoginViewController: UIViewController {
         
         fetchFBUserEndpointRequest.success = { (endpointRequest: EndpointRequest) in
             if let result = endpointRequest.processedResponseValue as? JSON {
-                User.currentUser.fillFields(data: result)
+                CurrentUser.currentUser.facebookID = result["user"]["fb_id"].string!
```

Comment:
if it sounds stupid when you say it, its probably a bad idea to use CurrentUser.currentUser

should be User.currentUser or System.currentUser